docs/guides/migrating-from-camunda-7/conceptual-differences.md

```diff
@@ -46,7 +46,21 @@ To migrate existing Connectors, consider the following options:
 - Use the [REST protocol Connector](components/connectors/protocol/rest.md) to leverage an out-of-the-box Connector.
 - Create a small bridging layer via custom [job workers](/components/concepts/job-workers.md).
 
-### Process solutions using Spring Boot
+### Multi-tenancy
+
+:::note
+[Multi-tenancy](/self-managed/concepts/multi-tenancy.md) is currently in development for Camunda 8.
```

Comment:
:thought_balloon: I assume we note this since we only support multi-tenancy for self-managed but not for SaaS/